_includes/footer.html

```diff
@@ -1,6 +1,8 @@
 </main>
 <footer class="mk-main-footer">
-  <p>&copy; 2019 metal3.io <a href="/privacy-statement.html">Privacy Statement</a></p>
+  <p>We are a <a href="https://cncf.io/">Cloud Native Computing Foundation</a> sandbox project.</p>
+  <p><img src="/assets/images/cncf-color.png"/></p>
+  <p>&copy; 2020 metal3.io <a href="/privacy-statement.html">Privacy Statement</a></p>
```

Comment:
I believe the copyright should be amended to adopt the [CNCF guidelines on copyright notices](https://github.com/cncf/foundation/blob/master/copyright-notices.md#copyright-notices).

Something like: Copyright The Metal³ Contributors

Reply:
good catch!